dasharo-compatibility/dmidecode.robot

```diff
@@ -34,7 +34,7 @@ DMI001.001 Verify the device serial number
     Login To Linux
     Switch To Root User
     Detect Or Install Package    dmidecode
-    ${out}=    Execute Linux Command    dmidecode -t bios | grep Serial
+    ${out}=    Execute Linux Command    dmidecode -t system | grep Serial
```

Comment:
The command was wrong, fixed according to test specification https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31L-smbios/#dmi001001-verify-the-serial-number